pkg/mons/mon-endpoints_test.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
nit: How about naming them mon_endpoints.go and mon_endpoints_test.go? At least in rook we have this pattern of underscores instead of dashes for go filenames.

Reply:
done